mobika2_tests/config/test_5_with_nav.yaml

```diff
@@ -0,0 +1,90 @@
+runner:
+  start_logging_offset: 2
+  verification_timeout: 0.25
+  goal_reach_timeout: 18
+   
+interpreters: 
+  goal:
+    frame_id: map
+    child_frame_id: goal
+    position:
+      x: -0.26
+      y: -1.88
+      z: 0
+    yaw: 1.57 # degree
+  goal_action: move_base # if "None", won't use action
+  wait_action: 5 # time for waiting for action available
+  rate: 100
+
+  obstacle_tf2_broadcaster_node:
+    goal:
+      child_frame_id: obstacle
+      frame_id: map
+      position:
+        x: -1
+        y: 0
+        z: 0
+      yaw: 1.57 # degree
```

Comment:
I will remove this part. We trust mavelmind will provide this frame

Reply:
Do you want to remove the obstacle_tf2_broadcaster_node completely?

Reply:
If we want to use this part, we need use code from ipa320/araig_test_stack#53. Either I remove this part, to you rebase your branch on this branch